webplugin/js/app/mck-app.js

```diff
@@ -28,8 +28,8 @@ function ApplozicSidebox() {
     var mck_external_scripts = [
         {
             name: 'applozic-min-js',
-            url: 'https://cdn.kommunicate.io/applozic/applozic.chat-6.1.1.min.js',
-            alternateUrl: MCK_STATICPATH + '/js/app/applozic.chat-6.1.1.min.js',
+            url: 'https://cdn.kommunicate.io/applozic/applozic.chat-6.2min.js',
```

Comment:
file name is not correct 6.2.min.js
get this corrected if it is like this on cdn also.

Also,  have you checked the URL. It is showing access denied for me.

Reply:
added the CDN url